packages/google-pay-integration/src/google-pay-paypal-commerce/google-pay-paypal-commerce-payment-strategy.ts

```diff
@@ -95,6 +95,12 @@ export default class GooglePayPaypalCommercePaymentStrategy extends GooglePayPay
             .Googlepay()
             .confirmOrder({ orderId, paymentMethodData: confirmOrderData });
 
+        if (status === 'PAYER_ACTION_REQUIRED') {
```

Comment:
i think it would be better to create `enum` type for google pay order confirmation statuses

Reply:
Done